source/images/settlement/images/repayments_flow_high_level.puml

```diff
@@ -36,8 +36,8 @@ end note
 api -[#007500]-> other : <b>INVESTMENT_REPAYMENT_INSTRUCTION_RECEIVED</b> webhook
 ... ...
 api -[#007500]-> other : <b>INVESTMENT_REPAYMENT_CLEARING</b> webhook (1/10)
-api -[#007500]-> other : INVESTMENT_REPAYMENT_CLEARING webhook (2/10)
-api -[#007500]-> other : ...
+api -[#007500]-> other : CASH_INTEREST_REPAYMENT_CLEARING webhook (2/10)
```

Comment:
I think this diagram would be better if line 39 is removed and added to line 38 above. So that it is either a <b> INVESTMENT_REPAYMENT_CLEARING </b>  or <b>CASH_INTEREST_REPAYMENT_CLEARING</b> webhook

Reply:
Modified.